js/router.js

```diff
@@ -390,7 +390,7 @@ module.exports = Backbone.Router.extend({
       bodyClass: 'onPage'
     });
 
-    app.appBar.setTitle(handle ? handle : options.userId || this.userModel.get('guid'));
+    app.setTitle(handle ? handle : options.userId || this.userModel.get('guid')); //router problem fix
```

Comment:
I think both of them(appBar and setTitle) are not necessary.
This code below works.
```
app(handle ? handle : options.userId || this.userModel.get('guid'));
```

But this doesn't make sense to me because app is an object.